src/Core/System/Resources/config/listing.xml

```diff
@@ -36,5 +36,14 @@
             <helpText>Products are marked as new, if their release date does not date back past the set number of days.</helpText>
             <helpText lang="de-DE">Produkte werden als neu markiert, wenn ihr Erscheinungsdatum nicht länger als die angegebene Anzahl an Tagen zurückliegt.</helpText>
         </input-field>
+
+        <input-field>
+            <name>productCountSteps</name>
+            <label>Selection of products per page</label>
+            <label lang="de-DE">Anzahlen der Produkte je Seite</label>
+            <helpText>Integers between 1 and 100 separated by comma</helpText>
```

Comment:
Why there should be a limit of 100? 🤔

Reply:
I stumbled upon a limitation of 100 somewhere but can't find it right now.

Reply:
I cannot see any limit. Just tested with `?limit=500` with correct result. It looks like the config `shopware.api.max_limit` is relevant :-)

Reply:
Maybe there should be a check `shopware.api.max_limit` to not exceeding it? 🤔 https://github.com/shopware/platform/blob/7704106f4627996de0689b4ea91db58b1687f9ff/src/Core/Content/Product/SalesChannel/Listing/ProductListingFeaturesSubscriber.php#L496-L505

Reply:
![grafik](https://user-images.githubusercontent.com/1133593/89184770-92555580-d599-11ea-98d2-5992f98624df.png)
I mean it is already checked :) but I know where you looked at.

Reply:
You're a stalker! :-)